packages/rancher-logging/4.10/generated-changes/overlay/templates/_generic_logging.yaml

```diff
@@ -25,6 +25,17 @@ spec:
     scaling:
       replicas: {{ .Values.fluentd.replicas }}
     {{- end }}
+    security:
+      securityContext:
+        allowPrivilegeEscalation: false
+        readOnlyRootFilesystem: false
+        capabilities:
+          drop:
+            - ALL
+      podSecurityContext:
+        fsGroup: 101
+        runAsUser: 100
```

Comment:
Any specific reason on the `100` for the user ID? The convention is that anything below a `1000` is reserved to system accounts, so the recommendation here would be to use at least `1000`.